lib/rspec/rails/configuration.rb

```diff
@@ -140,8 +142,16 @@ def filter_rails_from_backtrace!
       if defined?(ActiveJob)
         config.include RSpec::Rails::JobExampleGroup, :type => :job
       end
+
+      if defined?(ActionCable) && ::Rails::VERSION::STRING > '6'
+        config.include RSpec::Rails::ChannelExampleGroup, :type => :channel
+      end
+
+      if defined?(ActionMailbox)
```

Comment:
We should use the feature checks here and above...

Reply:
See https://github.com/rspec/rspec-rails/pull/2131